graphql/server.js

```diff
@@ -33,6 +37,41 @@ const limiter = new RateLimit({
   delayMs: 100, // by 1 second
 });
 
+const readEntities = (dir, ext) => {
+  return _.fromPairs(fs.readdirSync(dir)
+    .map(file => file.endsWith(ext) && file.substr(0, file.length - ext.length))
```

Comment:
sooo you first want to filter out files with not matching extension and then load them?
why can't you write:

```
  .filter(file => file.endsWith(ext))
  .map....
```

Now map and filter is in wrong order... i think